client/app/components/EmailSettingsWarning.jsx

```diff
@@ -16,16 +16,20 @@ export default function EmailSettingsWarning({ featureName, className, mode, adm
   }
 
   const message = (
-    <span>
+    <span id="sr-mail-description">
       Your mail server isn&apos;t configured correctly, and is needed for {featureName} to work.{" "}
       <HelpTrigger type="MAIL_CONFIG" className="f-inherit" />
     </span>
   );
 
   if (mode === "icon") {
     return (
-      <Tooltip title={message}>
-        <i className={cx("fa fa-exclamation-triangle", className)} />
+      <Tooltip title={message} placement="topRight" arrowPointAtCenter>
+        <i
+          className={cx("fa fa-exclamation-triangle", className)}
+          aria-label="Mail alert"
```

Comment:
What's the difference between using `aria-label` vs the previous `<span className="sr-only" />`?

Reply:
The idea is that labels are a bit simpler, but there shouldn't be much of one. Also, labels must only be added to interactive or page-semantic elements, which is the major reason the second approach was used elsewhere, as I find it a bit cumbersome.

Reply:
Ok, so specifically for this case, which is the same `<i>` used for an icon, is there a reason to use `aria-label` in here and not in the others (or the opposite way and not use the `<span>` in here specifically)? I'm mostly wondering what's the approach we should follow when adding an icon into our codebase

Reply:
Actually this one was sort of a mistake, because it makes no sense to have such text in icons, which should be hidden to SRs.